File: 백준/Bronze/2525. 오븐 시계/오븐 시계.rb
hours, minutes = gets.split.map(&:to_i)
timer = gets.to_i
addHours = 0
addMinutes = 0

addHours += (timer / 60)
addMinutes += (timer % 60)

hours += addHours
minutes += addMinutes

if minutes >= 60
    hours += 1
    minutes %= 60
end

if hours >= 24
    hours %= 24
end

puts "#{hours} #{minutes}"

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.c
#include <stdio.h>

int main() {
    int hours = 0, minutes = 0, timer = 0;

    scanf("%d %d\n%d", &hours, &minutes, &timer);

    int addHours = timer / 60;
    int addMinutes = timer % 60;

    hours += addHours;
    minutes += addMinutes;
    
    if(minutes >= 60) {
        hours += 1;
        minutes %= 60;
    }

    if(hours >= 24) {
        hours %= 24;
    }

    printf("%d %d", hours, minutes);

    return 0;
}